source/config/configSpec.py

```diff
@@ -83,6 +83,8 @@
 	showMessages = integer(0, 2, default=1)
 	# Timeout after the message will disappear from braille display
 	messageTimeout = integer(default=4, min=1, max=20)
+	# Rate for automatic scroll
+	autoScrollRate = integer(default=10, min=1, max=20)
```

Comment:
is 20 different rates too limiting? perhaps the interval step should be 0.1 or 0.5?

Reply:
Generally I read at 12 cells/sec or so. According to a rule used in Spain for captions, the recommended speed is 12 chars/sec or so too. I think that 20 is a lot, but may be that in some texts can be used.
We can use 30 if you want. When lines are short, I use the keystroke in the braille display to scroll forward manually.

Reply:
OK, then 20 may be enough, but I need to investigate how to allow 0.1 interval using seconds. Perhaps milliseconds wikk be better, or using a slider. I don't know another samples of float numbers with spin controls.

Reply:
I think an interval of 0.5 might be enough? We can always change it 0.1 later